README.md

```diff
@@ -23,6 +23,16 @@ This will:
 * Enforce mandatory TLS to some major email domains
 * Enforce minimum TLS versions to some major email domains
 
+## Using the STARTTLS Policy List
+
+To download and verify the most up-to-date version of the STARTTLS Policy List:
+```
+wget https://dl.eff.org/starttls-everywhere/policy.json
+wget https://dl.eff.org/starttls-everywhere/policy.json.asc
+gpg --recv-key B693F33372E965D76D55368616EEA65D03326C9D
```

Comment:
Has this key been uploaded to the servers yet? When I tried to fetch it from the keyservers, gpg said the key couldn't be retrieved...

Reply:
Nevermind--works now!

Reply:
Yeah, I uploaded it a couple of weeks ago: https://pgp.mit.edu/pks/lookup?op=get&search=0x16EEA65D03326C9D

Odd that it hasn't propagated fully. Going to look into this a bit more.